### PR TITLE
add relayer fee universalswap

### PR DESCRIPTION
**Pull request by haunv3**
- Add relayer fee universalswap

### CODE REVIEW

File: src/rest/api.ts [hunk 1]
```diff
@@ -451,6 +451,16 @@ async function fetchTaxRate(): Promise<TaxRateResponse> {
   }
 }
 
+async function fetchRelayerFee(): Promise<any> {
+  const ics20Contract = new CwIcs20LatestQueryClient(window.client, process.env.REACT_APP_IBC_WASM_CONTRACT);
+  try {
+    const { relayer_fees } = await ics20Contract.config();
```
Comment (tucq88) [inline src/rest/api.ts:457]: Why not use `camelCase` like other variables ?

Comment (haunv3) [reply src/rest/api.ts:457]: It is a key in the object so i used it always
Screen Shot 2023-10-12 at 16 28 37

Comment (tucq88) [reply src/rest/api.ts:457]: Then I would success to contract object with `Ics2Contract` type , then return properly with `Ics2Contract.relayer_fees`. It makes context clearer. 

But not very critical one. You can add comment `//REFACTOR:` for managing refactor list later.

**Comment by tucq88** [comment]
LGTM now, if you need anyone to review logic, feel free to add or merge it anyway.

**Review by ducphamle2 — APPROVED** [review]

**Pull request merged**